src/modules/get.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
```

Comment:
As Linters help us to write structured and cleaner code, it's really useful to use them

- Kindly as it a basic requirement to every project on Microverse, delete this comment and apply the fix to run the linters 💯